.github/workflows/deprecate.yml

```
@@ -0,0 +1,27 @@
+name: "Deprecate package"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'type version: x.y.z (without "v")'
+        required: true
+      reason:
+        description: "reason for deprecation"
+        required: true
+
+run-name: Deprecate version ${{ inputs.version }}
+
+jobs:
+  deprecate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
```

```suggestion
          node-version: 18
```
Даня в другом PR обновил до 18 – уже в `master`)